cg/services/order_validation_service/workflows/mutant/constants.py

```diff
@@ -4,7 +4,7 @@
 
 
 class MutantDeliveryType(Enum):
-    ANALYSIS_FILES = DataDelivery.FASTQ_ANALYSIS
+    ANALYSIS_FILES = DataDelivery.FASTQ
```

Comment:
mmm this one is polemic, see https://github.com/Clinical-Genomics/improve-order-flow/issues/94. It all depends on which is implemented first

Reply:
I guess, but currently fastq + analysis has no support. I could wait with the change and just let the test fixture have "No Delivery" though?

Reply:
I think the other will get merged first. I guess for now it is not very important, but just keep this in mind for later

Reply:
I will resolve for now and create an issue where we can track all the uncertainties that should be covered in the final tests

Reply:
WIP: https://github.com/Clinical-Genomics/improve-order-flow/issues/99